express/code/blocks/interactive-marquee/interactive-marquee.js

```diff
@@ -27,7 +27,8 @@ async function handleGenAISubmit(form, link) {
   const input = form.querySelector('input');
   if (!link || input.value.trim() === '') return;
   const mod = await import('../../scripts/branchlinks.js');
-  const genAILink = await mod.getTrackingAppendedURL(link).replace(promptTokenRegex, encodeURI(input.value).replaceAll(' ', '+'));
+  const trackingLink =  await mod.getTrackingAppendedURL(link);
```

Comment:
lint error. I think you can also fix this by just wrapping the await with parenthesis, but currently this is fine